### ISSUE TITLE
Switch seems to no longer be supported

### ISSUE THREAD

**Issue by 1gn0r4nd** [gitdagray/learn_react_router#2]
I get the following error:
export 'Switch' (imported as 'Switch') was not found in 'react-router-dom'
I think we need to upgrade this:
s/Switch/Routes/g

https://reactrouter.com/docs/en/v6/upgrading/v5

**Comment by gitdagray** [comment]
Yes, as noted in the video description, I have an updated tutorial for
React Router v6 using the same blog project here:
https://youtu.be/XBRLVRjZ3CQ
